modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -640,6 +637,22 @@ p.ddm-label {
 	word-wrap: break-word;
 }
 
+.field-base-flex-container {
+	display: flex;
+}
+
+.field-base-flex-element {
+	flex-grow: 1;
+}
+
+.field-base-label {
+	word-break: break-word;
+}
+
+.field-base-legend {
+	word-break: break-word;
+}
+
```

Comment:
@mateusgomes01, one important thing that I forgot to mention to you: new CSS classes should be added following alphabetic order. :wink: Can you please make sure that this commit follows this rule.